darts/tests/metrics/test_metrics.py

```diff
@@ -1539,3 +1568,265 @@ def helper_test_non_aggregate(self, metric, is_aggregate, val_exp=None):
 
         if val_exp is not None:
             assert (res == -1.0).all()
+
+    @pytest.mark.parametrize(
+        "config",
+        list(
+            itertools.product(
+                [
+                    # time dependent but with time reduction
+                    metrics.err,
+                    metrics.ae,
+                    metrics.se,
+                    metrics.sle,
+                    metrics.ase,
+                    metrics.sse,
+                    metrics.ape,
+                    metrics.sape,
+                    metrics.arre,
+                    metrics.ql,
+                    # time aggregates
+                    metrics.merr,
+                    metrics.mae,
+                    metrics.mse,
+                    metrics.rmse,
+                    metrics.rmsle,
+                    metrics.mase,
+                    metrics.msse,
+                    metrics.rmsse,
+                    metrics.mape,
+                    metrics.smape,
+                    metrics.ope,
+                    metrics.marre,
+                    metrics.r2_score,
+                    metrics.coefficient_of_variation,
+                    metrics.mql,
+                ],
+                [True, False],  # univariate series
+                [True, False],  # single series
+            )
+        ),
+    )
+    def test_metric_quantiles(self, config):
+        """Test output types and shapes for time aggregated metrics:
+        for single and multiple univariate or multivariate series, in combination
+        with different component and series reduction functions."""
+        np.random.seed(42)
+        metric, is_univar, is_single = config
+        params = inspect.signature(metric).parameters
+
+        n_comp = 1 if is_univar else 2
+
+        qs_all = [0.1, 0.5, 0.8]
```

Comment:
Is is intentional to have "asymmetric" quantiles (aka having 0.1 and 0.8 instead of 0.1 with 0.9 or 0.2 with 0.8)?

Reply:
Yes, in theory you can compute it on any quantile.